libs/sdk-core/src/chain.rs

```diff
@@ -75,13 +75,21 @@ impl AddressUtxos {
 }
 
 /// Gets unspent tx outputs. Specifically filters out inbound utxos that have been spent.
-pub(crate) fn get_utxos(address: String, transactions: Vec<OnchainTx>) -> Result<AddressUtxos> {
+/// If include_unconfirmed_spends is true, then the result won't include utxos that were spent
+/// in unconfirmed transactions.
+pub(crate) fn get_utxos(
+    address: String,
+    transactions: Vec<OnchainTx>,
+    include_unconfirmed_spends: bool,
+) -> Result<AddressUtxos> {
     // Calculate confirmed amount associated with this address
```

Comment:
```suggestion
```

IMO better to remove this comment, as the method doesn't only deal with confirmed utxos anymore.

Reply:
removed